pages/index.tsx

```diff
@@ -1,15 +1,42 @@
 import Head from 'next/head';
-import {Container, Row, Col, Jumbotron, Image, Carousel} from 'react-bootstrap';
-import {BiRecycle} from 'react-icons/bi';
+import { Container, Row, Col, Jumbotron, Image, Carousel, Accordion, Card, Button, AccordionContext, useAccordionToggle } from 'react-bootstrap';
+import { BiRecycle } from 'react-icons/bi';
 
 import etcStyles from '../styles/etc.module.css';
 
 import Navigation from '../components/navbar';
 import Footer from '../components/footer';
+import { useContext } from 'react';
+
+interface Toggle{
+  children: React.ReactNode,
+  eventKey: string,
+  callback?: Function
+}
+
+interface HomeContent{
+  children: React.ReactNode
+}
+
+const AwareToggle = ({children, eventKey, callback}: Toggle) => {
```

Comment:
nit: I would name the `callback` prop `onToggle` instead which is more descriptive.

Also, I would spell out "ContextAwareToggle" rather than shortening it to "AwareToggle". The use of the word "aware" is confusing by itself.